src/util/datablock/oo_datablock.h

```diff
@@ -27,3 +27,4 @@ void *DataBlock_AllocateItemOutOfOrder(DataBlock *dataBlock, uint64_t idx);
  * 1. Item distructor is not invoked in this call.
  * 2. This call does not decrease the number of items in the data block. */
 void DataBlock_MarkAsDeletedOutOfOrder(DataBlock *dataBlock, uint64_t idx);
```

Comment:
if the c file deleted why not delete this file too?

Reply:
clear separation of the API, but we could delete this one as-well, I just wanted to keep this API separated.